packages/patternfly-4/react-core/src/components/Nav/NavGroup.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Nav/nav';
 import { css } from '@patternfly/react-styles';
+import { NavVariants } from './NavVariants';
```

Comment:
You imported `NavVariants` but you're not using it.

Reply:
nice catch -- fixed. thank you!